### PR TITLE
Update README.md to use proper git clone url

### PR DESCRIPTION
**Pull request by a-dubs**
You know what they say. Better late than never ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,10 +14,10 @@ pip3 install pycloudlib
 
 Project's requirements.txt file can include pycloudlib as a dependency. Check out the [pip documentation](https://pip.readthedocs.io/en/1.1/requirements.html) for instructions on how to include a particular version or git hash.
 
-Install from latest master:
+Install from latest changes in `main` branch:
 
 ```shell
-git clone https://git.launchpad.net/pycloudlib
+https://github.com/a-dubs/pycloudlib.git
```
Comment (TheRealFalcon) [inline README.md:20]: We don't `a-dubs`'s version either :wink:

Comment (a-dubs) [reply README.md:20]: BRUH

**Comment by TheRealFalcon** [comment]
Don't forget the version bump either

File: README.md [hunk 2]
```diff
@@ -14,10 +14,10 @@ pip3 install pycloudlib
 
 Project's requirements.txt file can include pycloudlib as a dependency. Check out the [pip documentation](https://pip.readthedocs.io/en/1.1/requirements.html) for instructions on how to include a particular version or git hash.
 
-Install from latest master:
+Install from latest changes in `main` branch:
 
 ```shell
-git clone https://git.launchpad.net/pycloudlib
+https://github.com/canonical/pycloudlib.git
```
Comment (TheRealFalcon) [inline README.md:20]: Whoops...looks like we lost the `git clone` here now too

Comment (a-dubs) [reply README.md:20]: :sob: :sob: :sob: thank you

**Review by TheRealFalcon — APPROVED** [review]
LGTM!

**Pull request merged**